src/game/editor/mapitems/layer_tiles.cpp

```diff
@@ -162,13 +163,13 @@ void CLayerTiles::Render(bool Tileset)
 	if(!Tileset)
 	{
 		if(m_Tele)
-			m_pEditor->RenderTools()->RenderTeleOverlay(static_cast<CLayerTele *>(this)->m_pTeleTile, m_Width, m_Height, 32.0f);
+			m_pEditor->RenderTools()->RenderTeleOverlay(static_cast<CLayerTele *>(this)->m_pTeleTile, m_Width, m_Height, 32.0f, g_Config.m_ClTextEntitiesEditor);
 		if(m_Speedup)
-			m_pEditor->RenderTools()->RenderSpeedupOverlay(static_cast<CLayerSpeedup *>(this)->m_pSpeedupTile, m_Width, m_Height, 32.0f);
+			m_pEditor->RenderTools()->RenderSpeedupOverlay(static_cast<CLayerSpeedup *>(this)->m_pSpeedupTile, m_Width, m_Height, 32.0f, g_Config.m_ClTextEntitiesEditor);
 		if(m_Switch)
-			m_pEditor->RenderTools()->RenderSwitchOverlay(static_cast<CLayerSwitch *>(this)->m_pSwitchTile, m_Width, m_Height, 32.0f);
+			m_pEditor->RenderTools()->RenderSwitchOverlay(static_cast<CLayerSwitch *>(this)->m_pSwitchTile, m_Width, m_Height, 32.0f, g_Config.m_ClTextEntitiesEditor);
 		if(m_Tune)
-			m_pEditor->RenderTools()->RenderTuneOverlay(static_cast<CLayerTune *>(this)->m_pTuneTile, m_Width, m_Height, 32.0f);
+			m_pEditor->RenderTools()->RenderTuneOverlay(static_cast<CLayerTune *>(this)->m_pTuneTile, m_Width, m_Height, 32.0f, g_Config.m_ClTextEntitiesEditor);
```

Comment:
```suggestion
			m_pEditor->RenderTools()->RenderTuneOverlay(static_cast<CLayerTune *>(this)->m_pTuneTile, m_Width, m_Height, 32.0f, g_Config.m_ClTextEntitiesEditor ? OVERLAYRENDERFLAG_TEXT : 0);
```
Alternatively, make it obvious what flag you're setting.

Reply:
I added this as a render-flag, because this is how it's done for other similar functions like RenderTelemap, RenderSpeedupmap, etc. I believe, that there might come more flags in the future and it's a bit cleaner this way